install.sh

```
@@ -69,6 +116,68 @@ MANAGER_TARGET=$ROOT/bin/Wine4OfficeManager
 UNINSTALLER_TARGET=$ROOT/bin/wine4office-uninstall
 RUNNER_TARGET=$ROOT/runner
 
+is_active_install_pid() {
+    local pid=$1 owner executable
+    [[ $pid =~ ^[0-9]+$ && -d /proc/$pid ]] || return 1
+    owner=$(stat -c %u "/proc/$pid" 2>/dev/null) || return 1
+    [[ $owner == $EUID ]] || return 1
```

Shellcheck (0.11.0)

[warning] Quote the right-hand side of == in [[ ]] to prevent glob matching.

(SC2053)